Fix exit code of test runner
Has been reported to the developer via email
--- src/testcases/main.cpp.orig	2006-08-20 09:50:36.000000000 -0500
+++ src/testcases/main.cpp	2015-05-07 00:19:39.000000000 -0500
@@ -16,11 +16,12 @@
 
 #include <cppunit/extensions/TestFactoryRegistry.h>
 #include <cppunit/ui/text/TestRunner.h>
+#include <cstdlib>
 
 int main(int argc, char** argv) {
     CppUnit::TextUi::TestRunner runner;
     CppUnit::TestFactoryRegistry &registry = CppUnit::TestFactoryRegistry::getRegistry();
     runner.addTest( registry.makeTest() );
     bool wasSuccessful = runner.run( "", false );
-    return wasSuccessful;
+    return wasSuccessful ? EXIT_SUCCESS : EXIT_FAILURE;
 }
